src/main/kotlin/com/sourcegraph/cody/config/CodyAuthenticationManager.kt

```diff
@@ -109,21 +107,15 @@ class CodyAuthenticationManager :
 
   fun addAuthChangeListener(project: Project) {
```

Comment:
This method should be now part of `CodyWindowAdapter`, it does not make sense in `CodyAuthenticationManager`